config/database.yml

```diff
@@ -1,68 +1,30 @@
 
 development:
-  url: <%= Settings.database_url %>
+  primary:
+    url: <%= Settings.database_url %>
+  audit:
+    url: <%= "#{Settings.audit_db.url}_development" %>
+    migrations_paths: db/audit_migrate
 
-  # The specified database role being used to connect to postgres.
-  # To create additional roles in postgres see `$ createuser --help`.
-  # When left blank, postgres will use the default role. This is
-  # the same name as the operating system user that initialized the database.
-  #username: postgres
-
-  # The password associated with the postgres role (username).
-  #password:
-
-  # Connect on a TCP socket. Omitted by default since the client uses a
-  # domain socket that doesn't need configuration. Windows does not have
-  # domain sockets, so uncomment these lines.
-  #host: localhost
-
-  # The TCP port the server listens on. Defaults to 5432.
-  # If your server runs on a different port number, change accordingly.
-  #port: 5432
-
-  # Schema search path. The server defaults to $user,public
-  #schema_search_path: myapp,sharedapp,public
-
-  # Minimum log levels, in increasing order:
-  #   debug5, debug4, debug3, debug2, debug1,
-  #   log, notice, warning, error, fatal, and panic
-  # Defaults to warning.
-  #min_messages: notice
-
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
 test:
-  url: <%= Settings.test_database_url %><%= ENV['TEST_ENV_NUMBER'] %>
+  primary:
+    url: <%= Settings.test_database_url %><%= ENV['TEST_ENV_NUMBER'] %>
+  audit:
+    url: <%= "#{Settings.audit_db.url}_test#{ENV['TEST_ENV_NUMBER']}" %>
+    migrations_paths: db/audit_migrate
 
-# As with config/secrets.yml, you never want to store sensitive information,
-# like your database password, in your source code. If your source code is
-# ever seen by anyone, they now have access to your database.
-#
-# Instead, provide the password as a unix environment variable when you boot
-# the app. Read http://guides.rubyonrails.org/configuring.html#configuring-a-database
-# for a full rundown on how to provide these environment variables in a
-# production deployment.
-#
-# On Heroku and other platform providers, you may have a full connection URL
-# available as an environment variable. For example:
-#
-#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
-#
-# You can use this database configuration with:
-#
-#   production:
-#     url: <%= ENV['DATABASE_URL'] %>
-#
 production:
-  url: <%= Settings.database_url %>
-  connect_timeout: 5
-  variables:
-    statement_timeout: <%= ENV["STATEMENT_TIMEOUT"] || "60s" %>
-    lock_timeout: 15s
+  primary:
+    url: <%= Settings.database_url %>
+    connect_timeout: 5
+    variables:
+      statement_timeout: <%= ENV["STATEMENT_TIMEOUT"] || "60s" %>
+      lock_timeout: 15s
+  audit:
+    url: <%= "#{Settings.audit_db.url}" %>
```

Comment:
Do you think we need any of the connect/statement/lock timeouts here too?

Reply:
Possibly, we can add it here or in the database terraform itself. I think it makes sense to keep it there but we can decide and add it here if we want once we get the databases spun up.

Reply:
I talked with Josh and we are going to put the timeouts and other settings in the terraform.